Dockerfile.windows.devel

```diff
@@ -81,7 +81,7 @@ COPY . /src/
 
 WORKDIR /src/build
 
-RUN cmake -G "'Visual Studio 16 2019'" -DCMAKE_BUILD_TYPE=Release ../;
+RUN cmake -G "'Visual Studio 16 2019'" -DCMAKE_BUILD_TYPE=Release -DFLB_HTTP_SERVER=No ../;
```

Comment:
Can you add this flag to `cmake/windows.cmake`? This will enable
us to simply do like this:

```
$ cmake ..
$ cmake --build .
```